src/Umbraco.Core/Models/Mapping/MemberTabsAndPropertiesMapper.cs

```diff
@@ -237,6 +237,7 @@ internal static ContentPropertyDisplay GetLoginProperty(IMemberTypeService membe
             var userRoles = username.IsNullOrWhiteSpace() ? null : _memberService.GetAllRoles(username);
 
             // create a dictionary of all roles (except internal roles) + "false"
+            //TODO: use member role manager instead
```

Comment:
Nope, editing members in the back office is purely to do with our own member data. We don't want to mix this editing with the ability to extend/replace a role manager - which will only be used in limited cases.

Reply:
Gotcha, we're only using it for our own limited case not for the whole application. Thanks for explaining